package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "prometheus-gc-stats",
-  "version": "0.6.4",
+  "version": "0.6.5",
```

```suggestion
  "version": "0.6.4",
```

I'll bump this when making a release